src/components/Books.jsx

```diff
@@ -0,0 +1,61 @@
+import { useState } from 'react';
+import Card from './Card';
+import AddBook from './AddBook';
+
+function Books() {
+  const [books, setBook] = useState([
+    {
+      id: 1,
+      category: 'Action',
+      name: 'The Hunger Games',
+      director: 'Suzanne Collins',
+      completed: '64%',
+      chapter: 'Chapter 17',
+    },
+    {
+      id: 2,
+      category: 'Science Fiction',
+      name: 'Dune',
+      director: 'Frank Herbert',
+      completed: '8%',
+      chapter: 'Chapter 3: A Lession',
+    },
+    {
+      id: 3,
+      category: 'Economy',
+      name: 'Capital in the Century',
+      director: 'Suzanne Collins',
+      completed: '0%',
+      chapter: 'Introduction',
+    },
+  ]);
+
+  const addBook = (newBook) => {
+    setBook([newBook, ...books]);
+  };
+
+  const handleDelete = (nid) => {
+    setBook(books.filter((todo) => todo.id !== nid));
+  };
+
+  return (
+    <div>
+      {books.map((todo) => (
+        <Card
+          key={todo.id}
+          category={todo.category}
+          name={todo.name}
+          director={todo.director}
+          completed={todo.completed}
+          chapter={todo.chapter}
+          deleteBook={() => handleDelete(todo.id)}
+        />
+      ))}
```

Comment:
- Please follow the advice of the previous reviewers and make the lines from 43 to 53 a component. Like the previous reviewers advised, you can name it `BookList`. Doing this will help break your app down into more reusable component, which is one of the advantages of react. Please fix this 🙏

Reply:
@mckent05 I will make the required changes.